aiida/orm/utils/remote.py

```diff
@@ -84,8 +115,8 @@ def get_calcjob_remote_paths(pks=None, past_days=None, older_than=None, computer
 
     path_mapping = {}
 
-    for path, computer in query.all():
-        if path is not None:
-            path_mapping.setdefault(computer.uuid, []).append(path)
+    for calcjob_id, remote_data, computer in query.all():
+        path_mapping.setdefault(computer.uuid,
```

Comment:
Since you already fetch the `Computer` instance, I would return this in the `path_mapping`, especially since the CLI needs it and actually loads it again. This would be more efficient.

Reply:
I think the reason for using this approach is that there are many folders, there will be each `Computer` node for each match - so lots of `Computer` to be loaded. In reality we expect only as few distinct computers among many matches, so project the ids then load the unique computer can be more efficeint?

Reply:
or may be the query will just return a single `Computer` object but multiple times? I am not sure.

Reply:
> It is a good point, but then we should also just project just the UUID as the query is currently projecting the entire entity.
ahh, I did not spot that. Just fixed it.